### PR TITLE
Match CreatePartShadow & RemovePartShadow

### PR DESCRIPTION
**Pull request by Technoman589**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀 This PR matches 2 functions (+0.01%) on `us`.

**Review by ethteck — APPROVED** [review]

**Review by JoshDuMan — APPROVED** [review]

File: src/197F40.c [hunk 1]
```diff
@@ -2346,9 +2346,31 @@ ApiStatus SetPartAlpha(Evt* script, s32 isInitialCall) {
     return ApiStatus_DONE2;
 }
 
-INCLUDE_ASM(s32, "197F40", CreatePartShadow);
+ApiStatus CreatePartShadow(Evt* script, s32 isInitialCall) {
+    s32* args = script->ptrReadPos;
```
Comment (nanaian) [inline src/197F40.c:2350]:
```suggestion
    Bytecode* args = script->ptrReadPos;
```

File: src/197F40.c [hunk 2]
```diff
@@ -2346,9 +2346,31 @@ ApiStatus SetPartAlpha(Evt* script, s32 isInitialCall) {
...
+        actorID = script->owner1.actorID;
+    }
+
+    create_part_shadow(actorID, partIndex);
+    return ApiStatus_DONE2;
+}
 
-INCLUDE_ASM(s32, "197F40", RemovePartShadow);
+ApiStatus RemovePartShadow(Evt* script, s32 isInitialCall) {
+    s32* args = script->ptrReadPos;
```
Comment (nanaian) [inline src/197F40.c:2363]:
```suggestion
    Bytecode* args = script->ptrReadPos;
```

**Review by nanaian — CHANGES_REQUESTED** [review]
Just a small nit - use `Bytecode` for args even if typedef'd to s32

**Review by nanaian — APPROVED** [review]

**Pull request merged**